IdentityCore/src/pop_manager/MSIDDevicePopManager.m

```diff
@@ -169,8 +171,15 @@ - (NSString *)createSignedAccessToken:(NSString *)accessToken
         MSID_LOG_WITH_CTX(MSIDLogLevelInfo, nil, @"MSIDDevicePopManager: createSignedAccessToken nonce is empty");
     }
     
+    MSIDJwtAlgorithm alg = [[MSIDKeyOperationUtil sharedInstance] getJwtAlgorithmForKey:self.keyPair.privateKeyRef context:nil error:error];
+    if ([NSString msidIsStringNilOrBlank:alg])
```

Comment:
alg is MSIDJwtAlgorithm type. would it be safe to do this comparison?

Reply:
MSIDJwtAlgorithm is a typedef of NSString, so this is safe.